h/api/storage.py

```diff
@@ -140,3 +146,7 @@ def _prepare(annotation):
     transform.normalize_annotation_target_uris(annotation)
 
     transform.add_nipsa(annotation)
+
+
+def _postgres_enabled():
+    return pyramid.threadlocal.get_current_request().feature('postgres')
```

Comment:
It's a shame that you have to do the `threadlocal.get_current_request()` thing (I think of that as an advanced/internal Pyramid detail, better to avoid if you can), but I guess that is the right way, unless we want to add `request` as a param to each function in `storage`